src/transformers/pipelines/conversational.py

```diff
@@ -272,7 +272,7 @@ def __call__(self, conversations: Union[List[Dict], Conversation, List[Conversat
                 Whether or not to clean up the potential extra spaces in the text output.
             generate_kwargs:
                 Additional keyword arguments to pass along to the generate method of the model (see the generate method
-                corresponding to your framework [here](./model#generative-models)).
+                corresponding to your framework [here](https://huggingface.co/docs/transformers/v4.36.1/en/main_classes/text_generation)).
```

Comment:
We try to use local links instead of hardcoded links; can you replace your link by a local link to the doc you have in mind?

(probably `./main_classes/text_generation`)